src/main/AndroidManifest.xml

```diff
@@ -20,7 +19,7 @@
   <uses-permission android:name="com.android.launcher.permission.UNINSTALL_SHORTCUT"/>
 
   <!-- READ_EXTERNAL_STORAGE is required if users want to include photos from their phone  -->
-  <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE"/>
+  <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" android:maxSdkVersion="32"/>
```

Comment:
Can see the notes from the [Android 13 PR](https://github.com/medic/cht-android/pull/325#issuecomment-1693193306) for why this permission is not needed on 33+.